cvmfs/statistics_database.cc

```diff
@@ -155,11 +218,28 @@ StatisticsDatabase::~StatisticsDatabase() {
 
 
 int StatisticsDatabase::StoreStatistics(const perf::Statistics *statistics,
-                                        const std::string start_time,
-                                        const std::string finished_time) {
-  sqlite::Sql insert(this->sqlite_db(),
-                      PrepareStatement(statistics, start_time,
-                                                   finished_time));
+                                        const std::string &start_time,
+                                        const std::string &finished_time,
+                                        const std::string &command_name,
+                                        const swissknife::ArgumentList &args) {
+  std::string insert_statement;
+  if (command_name == "ingest" || command_name == "sync") {
+    insert_statement = PrepareStatementIntoPublish(statistics, start_time,
+                                                               finished_time);
+  } else if (command_name == "gc") {
+    std::string dry_run_ = "0";
+    const bool dry_run = (args.count('d') > 0);
```

Comment:
I think I don't need dry_run variable here.

Reply:
I'm thinking the same thing. Dry runs should be ignored - no need to save any stats in the SQLite db in this case.

Reply:
Indeed, dry runs should not be recorded in the database.

I would remove the dry_run field from the schema and only save actual GC runs, not the dry runs.